Lesson4/src/by/home/Main.java

```diff
@@ -0,0 +1,14 @@
+package by.home;
+
+import by.home.work.Computer;
```

Comment:
please remove PC part from this branch,. All PC related changes should belong to Lesson4 branch

Reply:
https://github.com/santooleg/tms2021Android/pull/8

Reply:
please remove this code from this branch